tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php

```diff
@@ -233,6 +234,25 @@ public function testCreateDropWorkspace(): void
         );
 
         $this->assertNull($datasets->getIterator()->current());
+
+        $cloudResourceManager = $this->clientManager->getCloudResourceManager($this->projectCredentials);
+        $actualPolicy = $cloudResourceManager->projects->getIamPolicy(
+            'projects/' . $projectId,
+            (new Google_Service_CloudResourceManager_GetIamPolicyRequest()),
+            []
+        );
+        $actualPolicy = $actualPolicy->getBindings();
+
+        $serviceAccRoles = [];
+        foreach ($actualPolicy as $policy) {
+            $membersString = json_encode($policy->getMembers());
+            assert(is_string($membersString));
+            if (stripos($membersString, 'deleted:serviceAccount:' . $wsServiceAccEmail) !== false) {
+                $serviceAccRoles[] = $policy->getRole();
+            }
+        }
+
+        $this->assertEmpty($serviceAccRoles);
```

Comment:
Ked si napr cherry picknem commit s testami do mastru tak to failne pretoze tam budu permissions s email ws service acc zacinajuceho na `deleted:serviceAccount:`